style.css

```diff
@@ -149,36 +154,30 @@ ul li:hover ul {
   margin-top: -15px;
 }
 
-.four:hover ~ hr {

```

Comment:
This link hover animation was partially replaced with another one for a refinement of the design